app/script/backup/BackupImportError.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+window.z = window.z || {};
+window.z.backup = z.backup || {};
+
+z.backup.importError = (() => {
+  class BackupImportError extends Error {
+    constructor(message = 'Something went wrong.') {
+      super(message);
+      Object.setPrototypeOf(this, BackupImportError.prototype);
+    }
+  }
+
+  class InvalidMetaDataError extends BackupImportError {
```

Comment:
Instead of getting a `z.backup.importError.InvalidMetaDataError` it would be nice to have a `z.backup.InvalidMetaDataError`.

Reply:
And should I merge all the `import` and `export` errors back to a single file (cause it won't be namespaced then)?